From afbb6434c51013aea0496b05bcdbb0d0535d0844 Mon Sep 17 00:00:00 2001
From: Matyas Selmeci <matyas@cs.wisc.edu>
Date: Tue, 24 Mar 2020 15:59:30 -0500
Subject: [PATCH] Fix 1.15->1.16 schema upgrade script

The host_channels_host_id_channel_id_key constraint doesn't exist in the
first place.

I'd change it to DROP CONSTRAINT IF EXISTS, but that's postgres 9 syntax
and RHEL 6 only has postgres 8.
---
 docs/schema-upgrade-1.15-1.16.sql | 1 -
 1 file changed, 1 deletion(-)

diff --git a/docs/schema-upgrade-1.15-1.16.sql b/docs/schema-upgrade-1.15-1.16.sql
index 6c0f850..98d70d6 100644
--- a/docs/schema-upgrade-1.15-1.16.sql
+++ b/docs/schema-upgrade-1.15-1.16.sql
@@ -59,6 +59,5 @@ ALTER TABLE host_channels ADD CONSTRAINT active_revoke_sane CHECK (
                                          OR (active IS NOT NULL AND revoke_event IS NULL AND revoker_id IS NULL));
 ALTER TABLE host_channels ADD PRIMARY KEY (create_event, host_id, channel_id);
 ALTER TABLE host_channels ADD UNIQUE (host_id, channel_id, active);
-ALTER TABLE host_channels DROP CONSTRAINT host_channels_host_id_channel_id_key;
 
 COMMIT;
-- 
2.6.3

